test/compflags/ferguson/check_ieee_float.chpl

```diff
@@ -0,0 +1 @@
+writeln("Hi");
```

Comment:
Any reason for having 2 tests, rather than one with 2 compopts? 


Not sure if I think this is actually a good idea or if it adds much, just throwing it out there: 

What do you think of having these tests through `--savec` and grep the Makefile to make sure the right thing is there? And maybe add compopts for `--ieee-float --no-ieee-float`, and `--no-ieee-float --ieee-float` variants ?